.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,370 @@
+    - if:
+      - payloadType: Pull_Request_Review
+      - isAction:
+          action: Submitted
+      - isReviewState:
+          reviewState: Changes_requested
+      then:
+      - addLabel:
+          label: 'needs author feedback :memo:'
+      description: Add "needs author feedback" label to pull requests when changes are requested
+    - if:
+      - payloadType: Pull_Request
+      - isActivitySender:
+          issueAuthor: True
+      - not:
+          isAction:
+            action: Closed
+      - hasLabel:
+          label: 'needs author feedback :memo:'
+      then:
+      - removeLabel:
+          label: 'needs author feedback :memo:'
+      - addLabel:
+          label: 'needs attention :wave:'
+      - addReply:
+          reply: "This PR has been marked as \"needs attention \U0001F44B\" and awaiting a response from the team. "
+      description: Remove needs author feedback label when the author responds to a pull request
+    - if:
+      - payloadType: Pull_Request
+      - not:
+          isAction:
+            action: Closed
+      - hasLabel:
+          label: 'no-recent-activity :chart_with_downwards_trend:'
+      then:
+      - removeLabel:
+          label: 'no-recent-activity :chart_with_downwards_trend:'
+      description: Remove no recent activity label from pull requests
+    - if:
+      - payloadType: Issue_Comment
+      - hasLabel:
+          label: 'no-recent-activity :chart_with_downwards_trend:'
+      then:
+      - removeLabel:
+          label: 'no-recent-activity :chart_with_downwards_trend:'
+      description: Remove no recent activity label when a pull request is commented on
+    - if:
+      - payloadType: Pull_Request_Review
+      - hasLabel:
+          label: 'no-recent-activity :chart_with_downwards_trend:'
+      then:
+      - removeLabel:
+          label: 'no-recent-activity :chart_with_downwards_trend:'
+      description: Remove no recent activity label when a pull request is reviewed
+    - if:
+      - payloadType: Pull_Request
+      - hasLabel:
+          label: 'auto merge :zap:'
+      then:
+      - enableAutoMerge:
+          mergeMethod: Merge
+      description: Automatically merge pull requests
+    - if:
+      - payloadType: Pull_Request
+      - labelRemoved:
+          label: 'auto merge :zap:'
+      then:
+      - disableAutoMerge
+      description: Automatically merge pull requests
+    - if:
+      - payloadType: Issue_Comment
+      then:
+      - cleanEmailReply
+      description: 'Remove Quoted Email Content '
+    - if:
+      - payloadType: Issue_Comment
+      - isAction:
+          action: Created
+      - not:
+          isActivitySender:
+            issueAuthor: True
+      - hasLabel:
+          label: 'needs attention :wave:'
+      - isOpen
+      then:
+      - removeLabel:
+          label: 'needs attention :wave:'
+      description: Remove Needs Attention label

```

Comment:
The previous configuration was limited to pull requests via the `"eventType": "pull_request"` configuration. This will bring back that condition. /cc @JohannesLampel

```suggestion
    - if:
      - payloadType: Issue_Comment
      - isPullRequest
      - isAction:
          action: Created
      - not:
          isActivitySender:
            issueAuthor: True
      - hasLabel:
          label: 'needs attention :wave:'
      - isOpen
      then:
      - removeLabel:
          label: 'needs attention :wave:'
      description: Remove Needs Attention label
```